lib/curl.c

```diff
@@ -244,11 +244,13 @@ curl_off_t curl_get_size(const char *src)
 
     /* get the size */
     curl_easy_setopt(c, CURLOPT_URL, src);
-    curl_easy_setopt(c, CURLOPT_NOBODY, 1);
+    curl_easy_setopt(c, CURLOPT_NOBODY, 1L);
     curl_easy_setopt(c, CURLOPT_FAILONERROR, true);
+    curl_easy_setopt(c, CURLOPT_FOLLOWLOCATION, 1L);
```

Comment:
Consider setting a maximum number of redirects too, the default is unlimited:

```
curl_easy_setopt(c, CURLOPT_MAXREDIRS, 10L);
```

Was always a bit annoyed FOLLOWLOCATION is not the default, I've had a few things break after something is moved to a new URL and my script doesn't have `curl -L`.

Reply:
Oh nice, I didn't know that.  Yeah, definitely something worth setting.